dx/tests/conftest.py

```diff
@@ -0,0 +1,71 @@
+import random
+import string
+
+import pandas as pd
+import pytest
+from IPython.core import formatters
+from IPython.core.formatters import BaseFormatter
+
+
+@pytest.fixture
+def pandas_formatter() -> BaseFormatter:
+    class PandasTableSchemaFormatter(BaseFormatter):
+        print_method = "_repr_data_resource_"
+        _return_type = (dict,)
+
+    return PandasTableSchemaFormatter()
+
+
+@pytest.fixture
+def ipython_display_formatters(pandas_formatter: BaseFormatter) -> dict:
+    """
+    Creates mock ipython display formatters for standard media types
+    and the pandas data resource media type.
+    """
+    return {
+        "application/pdf": formatters.PDFFormatter(),
+        "application/javascript": formatters.JavascriptFormatter(),
+        "application/json": formatters.JSONFormatter(),
+        "application/vnd.dataresource+json": pandas_formatter,
+        "text/html": formatters.HTMLFormatter(),
+        "text/latex": formatters.LatexFormatter(),
+        "text/markdown": formatters.MarkdownFormatter(),
+        "text/plain": formatters.PlainTextFormatter(),
+        "image/jpeg": formatters.JPEGFormatter(),
+        "image/png": formatters.PNGFormatter(),
+        "image/svg+xml": formatters.SVGFormatter(),
+    }
```

Comment:
You don't have to but I think it's fine to reduce the total set here in the tests since you only need `text/html`, `application/vnd.dataresource+json`, `text/plain`, and the dex media type.